ansible/roles/sufia/tasks/main.yml

```diff
@@ -89,6 +89,8 @@
       mode: "0770"
     - path: "{{ project_app_root }}/db"
       mode: "0770"
+    - path: "{{ project_app_root }}/public/uploads"
```

Comment:
Note that `config/initializers/sufia.rb` for both `data-repo` and `iawa` currently contain this:
```
config.upload_path = ->() { Rails.root + 'tmp' + 'uploads' }
```

So, that resultant path is the one that needs to be writeable.  From where in the application code are you getting the path `{{ project_app_root }}/public/uploads`?